packages/pagebuilder/lib/ContentTypes/DynamicBlock/dynamicBlock.shimmer.js

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import { arrayOf, shape, string, object } from 'prop-types';
+import defaultClasses from './dynamicBlock.shimmer.module.css';
+import { useStyle } from '@magento/venia-ui/lib/classify';
+import Shimmer from '@magento/venia-ui/lib/components/Shimmer';
+
+/**
+ * Page Builder Dynamic Block Shimmer component.
+ *
+ * @typedef DynamicBlockShimmer
+ * @kind functional component
+ *
+ * @param {props} props React component props
+ *
+ * @returns {React.Element} A React component that displays a Dynamic Block Shimmer.
+ */
+const DynamicBlockShimmer = props => {
+    const classes = useStyle(defaultClasses, props.classes);
+    const {
+        border,
+        borderWidth,
+        marginTop,
+        marginRight,
+        marginBottom,
+        marginLeft,
+        minHeight,
+        paddingTop,
+        paddingRight,
+        paddingBottom,
+        paddingLeft,
+        cssClasses = []
+    } = props;
+
+    const rootStyles = {
+        marginTop,
+        marginRight,
+        marginBottom,
+        marginLeft,
+        minHeight
+    };
+
+    const wrapperStyles = {
+        border,
+        borderWidth,
+        paddingTop,
+        paddingRight,
+        paddingBottom,
+        paddingLeft
+    };
+
+    // Only render shimmer if min-height specified
+    return minHeight ? (
+        <div className={classes.parent} style={rootStyles}>
+            <Shimmer
+                aria-live="polite"
+                aria-busy="true"
```

Comment:
Aria labels should also be added to the main component to indicate the block has finished loading.
You can refer to Banner component for an example.

Reply:
Moved aria labels to parent element.
Also created https://jira.corp.magento.com/browse/PWA-2620 to refactor `aria-busy` attributes as a global element.
cc @JustinConabreeAbsolunet